### PR TITLE
Replace contacts with identities

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Functions like get_or_create_contact are used to pull identities of multiple players in the system. Naming identity would simplify things.

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps Hi boss ;)  would you also want any variable names containing 'contact' to also be changed to 'identity' for consistency sake..?  
I am correct in assuming that you want ALL function names containing 'contact' to be changed, right?

**Comment by NicolaasDieBaas** [comment]
ALL functions only being three... ;)  so I guess you could ignore one of my questions above..
1. subscribe_contact
2. get_active_subscription_by_contact
3. get_active_subscriptions_by_contact

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas Yes to the functions. The variable names just need to be clear on 'who' the identity is, so the role player - and internally inside a util function like `subscribe_identity` needs to be `identity`. Make sense?

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps Yes!  All functions to be renamed.  Variables to be evaluated and renamed as necessary.

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: src/utils.js [hunk 1]
```diff
@@ -277,41 +277,41 @@ go.utils = {
         return go.utils
             .service_api_call("identities", "post", null, payload, 'identities/', im)
             .then(function(json_post_response) {
-                var contact_created = json_post_response.data;
-                // Return the contact
-                return contact_created;
+                var identity_created = json_post_response.data;
+                // Return the identity
+                return identity_created;
```
Comment (gsvr) [inline src/utils.js]: just looking at the function, let's not create the var for no reason but rather just return json_post_response.data.

File: src/utils.js [hunk 2]
```diff
@@ -277,41 +277,41 @@ go.utils = {
...
             });
     },
 
-    // Gets a contact if it exists, otherwise creates a new one
+    // Gets a identity if it exists, otherwise creates a new one
```
Comment (gsvr) [inline src/utils.js]: _an_ identity :)

File: src/utils.js [hunk 3]
```diff
@@ -277,41 +277,41 @@ go.utils = {
...
     get_or_create_identity: function(address, im, operator_id) {
         if (address.msisdn) {
             address.msisdn = go.utils
                 .normalize_msisdn(address.msisdn, im.config.country_code);
         }
         return go.utils
-            // Get contact id using msisdn
+            // Get identity id using msisdn
```
Comment (gsvr) [inline src/utils.js]: replace msisdn with address

**Comment by gsvr** [comment]
:+1: when travis is happy

**Pull request merged**